api/src/main/java/openconsensus/internal/Utils.java

```diff
@@ -129,6 +130,22 @@ public static void checkIndex(int index, int size) {
     }
   }
 
+  /**
+   * Throws a {@link NullPointerException} if any of the map elements is null.
+   *
+   * @param map the argument map to check for null.
+   * @param errorMessage the message to use for the exception. Will be converted to a string using
+   *     {@link String#valueOf(Object)}.
+   */
+  public static <K, V> void checkMapElementNotNull(
```

Comment:
Should it be public?

Reply:
Yes, otherwise other packages won't be able to use it.